bundles/org.openhab.binding.modbus.helioseasycontrols/src/main/resources/OH-INF/i18n/helioseasycontrols_de.properties

```diff
@@ -0,0 +1,204 @@
+# binding
+binding.modbus.name = HeliosEasyControls Binding
+binding.modbus.description = Dieses Binding erlabut die Integration von Helios KWL Ger�ten mit easyControls via Modbus TCP
+
+# thing type
+thing-type.modbus.helios-easycontrols.label = Helios KWL
+thing-type.modbus.helios-easycontrols.description = Dieses Binding erlabut die Integration von Helios KWL Ger�ten mit easyControls via Modbus TCP
```

Comment:
Is this copied from the binding's description by intention?

Reply:
I've aligned this now better with the original English text.